src/main/scala/lms/macros/CustomManifest.scala

```diff
@@ -0,0 +1,5 @@
+package lms.macros
+
+trait CustomManifest[T] extends Manifest[T] {
```

Comment:
That's not what I meant. And also why the runtime class would be an Int?
And why is that in macro?

Reply:
fixed :)